src/pages/requests/[requestId]/conversation/[conversationId]/index.svelte

```diff
@@ -1,5 +1,5 @@
 <script>
-import Request from '../../../../../components/Request.svelte'
```

Comment:
this is where this change really shines

Reply:
Right?! Also when copying or moving code from one level of hierarchy to another.